modules/line_item/src/Entity/CommerceLineItem.php

```diff
@@ -146,7 +146,7 @@ public function getOwner() {
    * {@inheritdoc}
    */
   public function getOwnerId() {
-    return $this->get('uid')->target_id;
+    return $this->get('uid')->targetId;
```

Comment:
We don't change property names, they're in the database (and this one is provided by core).